src/GitHub/Data/Webhooks/Payload.hs

```diff
@@ -660,6 +667,32 @@ instance FromJSON HookIssue where
       <*> o .:? "closed_at"
       <*> o .: "body"
 
+instance ToJSON HookIssue where
+  toJSON hookIssue = object
+    [ "url"           .= whIssueUrl hookIssue
+    , "labels_url"    .= whIssueLabelsUrl hookIssue
+    , "comments_url"  .= whIssueCommentsUrl hookIssue
+    , "events_url"    .= whIssueEventsUrl hookIssue
+    , "html_url"      .= whIssueHtmlUrl hookIssue
+    , "id"            .= whIssueId hookIssue
+    , "number"        .= whIssueNumber hookIssue
+    , "title"         .= whIssueTitle hookIssue
+    , "user"          .= whIssueUser hookIssue
+    , "labels"        .= whIssueLabels hookIssue
+    , "state"         .= whIssueState hookIssue
+    , "locked"        .= whIssueIsLocked hookIssue
+    , "assignee"      .= whIssueAssignee hookIssue
+    , "milestone"     .= whIssueMilestone hookIssue
+    , "comments"      .= whIssueCommentCount hookIssue
+    , "created_at"    .= whIssueCreatedAt hookIssue
+    , "updated_at"    .= whIssueUpdatedAt hookIssue
+    , "closed_at"     .= (case whIssueClosedAt hookIssue of
```

Comment:
Does it make sense to return an empty string if there is no value for `closed_at` - i.e. it is `Nothing`?

Instead, how about about letting it possibly be `null` via `fmap toJSON (whIssueClosedAt hookIssue)` ?

Reply:
Sounds good. I'll do that